apps/webapp/shared/responsive-image.tsx

```diff
@@ -0,0 +1,17 @@
+// https://nextjs.org/docs/messages/next-image-upgrade-to-13
+import Image from 'next/legacy/image';
+import { cn } from 'ui/lib/utils';
+
+interface ResponsiveImageProps {
+  src: string;
+  alt: string;
+  className?: string;
+}
+
+export const ResponsiveImage = ({ src, alt, className }: ResponsiveImageProps) => {
+  return (
+    <div className={cn('relative', className)}>
+      <Image src={src} alt={alt} layout='fill' objectFit='cover' />
+    </div>
+  );
+};
```

Comment:
Why is it called `ResponsiveImage`? This may need some code comments/documentation. 

With `objectFit='cover'` isn't there a chance things could get cut off?

Reply:
it's just a wrapper! maybe call it ImageWrapper?

Reply:
But even `ImageWrapper` is too generic. This is a special kind of image wrapper that does ObjectFit. Perhaps something like: `FilledImage`.